src/Ucommerce.Avenue.Umbraco/Controllers/ProductController.cs

```diff
@@ -51,33 +53,35 @@ private ActionResult RenderView(bool addedToBasket)
                 Tax = new Money(unitPrice * taxRate, currencyIsoCode).ToString(),
                 Price = new Money(unitPrice * (1.0M + taxRate), currencyIsoCode).ToString()
             };
-            
+
             if (!string.IsNullOrEmpty(currentProduct.PrimaryImageUrl))
             {
                 productViewModel.ThumbnailImageUrl = currentProduct.PrimaryImageUrl;
             }
 
-            productViewModel.Properties = MapProductProperties(currentProduct);
+            var variants = CatalogLibrary.GetVariants(currentProduct);
+
+            productViewModel.Properties = MapProductProperties(variants);
 
             if (currentProduct.ProductFamily)
             {
-                // TODO:
-                // productViewModel.Variants = MapVariants(currentProduct.Variants);
+                productViewModel.Variants = MapVariants(variants);
             }
 
-            bool isInBasket = TransactionLibrary.GetBasket(true).OrderLines.Any(x => x.Sku == currentProduct.Sku);
```

Comment:
Is it good idea to remove this? It serves as a toggle for a UI element (notification "banner").

Reply:
I have to, unfortunately, because the TransactionLib is using the old order context which uses the old transaction library. Which is out of sync because of the new url structure.

Reply:
Alright, let's see if we can fix that as part of new issue.